<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookDetails</class>
 <widget class="QWidget" name="BookDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_bookname1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>29</y>
     <width>500</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>1230</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookphoto">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>200</width>
     <height>330</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_intro">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>500</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>简介</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_intro_txt">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>540</y>
     <width>2200</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookname2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_author">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>作者</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_publish">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>出版社</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_ISBN">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>ISBN</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_collect">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>收藏数</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_grade">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>评分</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_remain">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>剩余/总库存</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_publishtime">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>320</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>出版时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_classify">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>分类</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookposition">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>440</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>图书位置</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_historyborrow">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>历史借阅数</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_classify_txt">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>260</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_remain_txt">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>350</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_publishtime_txt">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>320</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_grade_txt">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>290</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_historyborrow_txt">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>410</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_ISBN_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>230</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookposition_txt">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>440</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_collect_txt">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>380</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_publish_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>200</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_author_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>170</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_star">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>200</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>收藏+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_borrow">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>290</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>借阅+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reserve">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>380</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>预约+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
